build.xml

```diff
@@ -84,7 +84,7 @@
         value="${date.english} ${time.english}" />
 
     <!-- version is used in the jar manifest & in package.xml -->
-    <property name="version" value="9.8build"/>
+    <property name="version" value="9.9build"/>
```

Comment:
The original request was to fix (remove) the steps which were causing the build to fail because of the push to ECS at CSUS.  Changing the build number wasn't part of that, and we haven't made a decision on what should be "9.9" and what should be part of an updated 9.8 release.